.github/workflows/ci.yml

```diff
@@ -0,0 +1,13 @@
+name: CI
+
+on: [push, pull_request]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v3
+      - run: yarn install
+      - run: yarn test
```

Comment:
wouldn't this make it always fail for now? I guess setting it up would be better than not having it until we need it.

Reply:
yes, I'm following with a PR right after